client/src/App.js

```diff
@@ -4,8 +4,9 @@ import About from "./pages/About";
 import Donations from "./Donations";
 import ThankYou from "./pages/ThankYou";
 import Home from "./pages/Home";
-import About from "./pages/About";
```

Comment:
Hi @Monika39-maker, if we delete line 7 the /about page will not render

Reply:
so, leave it there,pls

Reply:
I might have mistakenly deleted it when solving merge request